src/game/client/econ/econ_trading.cpp

```diff
@@ -544,7 +544,7 @@ const char* UniverseToCommunityURL( EUniverse universe )
 	default: // return public if we don't have a better guess.
 	case k_EUniversePublic: return "https://steamcommunity.com";
 	case k_EUniverseBeta: return "https://beta.steamcommunity.com";
-	case k_EUniverseDev: return "https://localhost/community";
+	case k_EUniverseDev: return "http://localhost/community";
```

Comment:
question is why it was https in the first place. If the local server has a self-signed cert then this might even break support for local tests and development

Reply:
Good point.

I've looked for dev universe usage over the code base and found other places use ```http://``` for localhost. So I guessed as there are much more places with ```http://``` for localhost, it is the valid one. 

I may be wrong and all places should be changed to ```https://``` thought.